src/main/java/com/sap/sgs/phosphor/fosstars/nvd/NVD.java

```diff
@@ -91,36 +94,84 @@ String location() {
   }
 
   /**
-   * Returns a list of JSON files downloaded from the NVD.
+   * Returns content of the downloaded database.
    *
-   * @return A list of downloaded JSON files.
+   * @return A list of input streams that contain contend of NVD.
    * @throws IOException If something went wrong.
    */
-  List<String> jsonFiles() throws IOException {
-    try (Stream<Path> walk = Files.walk(Paths.get(downloadDirectory))) {
+  public List<InputStream> contents() throws IOException {
+    Path downloadDirectoryPath = Paths.get(downloadDirectory);
+    if (!Files.exists(downloadDirectoryPath)) {
+      return Collections.emptyList();
+    }
+
+    try (Stream<Path> walk = Files.walk(downloadDirectoryPath)) {
       return walk
           .filter(Files::isRegularFile)
           .filter(path -> path.toString().contains(NVD_FEED_VERSION))
           .filter(path -> path.toString().endsWith(".json"))
-          .map(Path::toString)
+          .map(NVD::open)
           .collect(Collectors.toList());
     }
   }
 
   /**
-   * Looks for NVD entries for a specific vendor and product.
+   * Creates an input stream for a file.
+   *
+   * @param file The file.
+   * @return The input stream.
+   */
+  private static InputStream open(Path file) {
```

Comment:
Do you think we need some Utils class to handle such methods
- Read file
- Do a HTTP call

A clean-up task?

Reply:
I am not sure about it.